test/e2e/tests/test_monitoring_schedule.py

```diff
@@ -0,0 +1,219 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+"""Integration tests for the SageMaker MonitoringSchedule API.
+"""
+
+import time
+import pytest
+import logging
+
+from e2e import service_marker, create_sagemaker_resource, wait_for_status
+from e2e.replacement_values import REPLACEMENT_VALUES
+from e2e.common.fixtures import (
+    xgboost_churn_data_quality_job_definition,
+    xgboost_churn_endpoint,
+)
+from acktest.k8s import resource as k8s
+from acktest.resources import random_suffix_name
+
+RESOURCE_PLURAL = "monitoringschedules"
+
+# Access variable so it is loaded as a fixture
+_accessed = xgboost_churn_data_quality_job_definition, xgboost_churn_endpoint
+
+
+@pytest.fixture(scope="module")
+def xgboost_churn_data_quality_monitoring_schedule(
+    xgboost_churn_data_quality_job_definition,
+):
+    (_, job_definition_resource) = xgboost_churn_data_quality_job_definition
+
+    job_definition_name = job_definition_resource["spec"].get("jobDefinitionName")
+
+    monitoring_schedule_name = random_suffix_name("monitoring-schedule", 32)
+
+    replacements = REPLACEMENT_VALUES.copy()
```

Comment:
Hmm it would be cool to just pass the replacement values and it merges it into `REPLACEMENT_VALUES` inside `create_sagemaker_resource`.

Reply:
will try this out in a separate PR